janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -745,29 +807,54 @@ private String getDeletionScript(KeyInformation.IndexRetriever information, Stri
                 case LIST:
                     final String jsValue = convertToJsType(deletion.value, compat.scriptLang(), Mapping.getMapping(keyInformation));
                     String index = INDEX_NAME + i++;
-                    script.append("def ")
+                    script.append("if (ctx._source.containsKey(\"")
+                        .append(deletion.field)
+                        .append("\")) {")
+                        .append("def ")
```

Comment:
Please, combine all predefined strings into a single string where possible. For example, `.append("\")) {").append("def ")` should be `.append("\")) {def ")`

Reply:
Done